erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -138,6 +138,7 @@ def on_submit(self):
 			self.update_against_document_in_jv()
 
 		self.update_time_sheet(self.name)
+		update_company_monthly_sales(self.company)
```

Comment:
don't call this after every submit - this will be expensive for large data steps... let targets be calculated only by the scheduler.